pkg/ddl/ingest/backend_mgr.go

```diff
@@ -153,7 +154,13 @@ func (m *litBackendCtxMgr) Register(
 		return nil, err
 	}
 
-	bcCtx := newBackendContext(ctx, jobID, bd, cfg, defaultImportantVariables, m.memRoot, m.diskRoot, etcdClient)
+	physical, logical, err := bd.GetTS(ctx)
+	if err != nil {
+		return nil, err
+	}
+	curTS := oracle.ComposeTS(physical, logical)
```

Comment:
this value should be saved in checkpoint, so we won't skip checking when failover

Reply:
maybe use DDL job RealStartTS?

Reply:
https://github.com/pingcap/tidb/pull/55588/commits/1eacf7d510730d8cb635700b77338ab576d8d565